tests/test_butlerUtils.py

```diff
@@ -64,6 +66,17 @@
 )
 
 
+def custom_show(*args, **kwargs):
+    pass
+
+
+# Override the show function with our custom one
+# to avoid getting the next Warning:
+# UserWarning: FigureCanvasAgg is non-interactive, and thus cannot be shown
+# plt.show()
+plt.show = custom_show
```

Comment:
As with the other review, I wonder if we can just remove the calls to `.show()` instead.

Reply:
So remove the show inside the code? The show() is not part from the test but from the function in the library. I guess you mean returning the matplotlib Plot and then the uplayer call the show() from it, (but that is a big change)

Reply:
I think that's an acceptable change - I don't think any of this library code should be calling `show()`, and so I think removing it is fine, and if things upstream break, that fixing them there is the right way to do things.